azure-pipelines/1esjobs.yml

```diff
@@ -14,7 +14,7 @@ stages:
               outputs:
                 - output: pipelineArtifact
                   targetPath: $(build.artifactstagingdirectory)/build/${{ job.name }}
-                  artifactName: Build
+                  artifactName: Build ${{ job.name }}

```

Comment:
Consider `build_${{ job.name }}` to avoid spaces

Reply:
It didn't mind the spaces and I think it looks better: 
![image](https://github.com/microsoft/vscode-azuretools/assets/36966225/ca63623a-a99c-4b8d-82ee-dadc28639a37)